src/rdf_triples.erl

```diff
@@ -158,6 +158,12 @@ ns_compact_1(Pred, {Ns, Uri, Next}) ->
             ns_compact_1(Pred, maps:next(Next))
     end.
 
+%% @doc Normalize namespaces that are known to have been renamed.
+-spec ns_normalize(Pred) -> NewPred
+    when Pred :: binary(),
+         NewPred ::binary().
+ns_normalize(<<"http://schema.org/", P/binary>>) -> <<"https://schema.org", P/binary>>;
+ns_normalize(Pred) -> Pred.
```

Comment:
Good to centrally manage renames, as we can expect more I guess 👍

Reply:
Yes, I think we will see enough variations.  Especially this kind of http/https switches.